comments/stakeholder_annotations.md

```diff
@@ -0,0 +1,46 @@
+## Objective 
+
+Direct users through an SMP-native painpoint/stakeholder annotation process that allows them to respond to map content. The input data generates an output table with the following structure, accessed via API:
+
+| Participant | Object | Category | Comment |
+|-------------|--------|----------|---------|
+| Joe Bloggs | Example Painpoint | Surprising | null |
+| Joe Bloggs | Example Stakeholder | Question | "blahblahblah" |
```

Comment:
Might be worth being explicit about how the non-iri forms for our first three columns are derived from the annotation form.